rs/src/management/http_client.rs

```diff
@@ -67,8 +79,12 @@ impl TunnelManagementClient {
     ) -> HttpResult<Vec<Tunnel>> {
         let url = self.build_uri(Some(cluster_id), TUNNELS_API_PATH);
         let request = self.make_tunnel_request(Method::GET, url, options).await?;
-        self.execute_json("list_cluster_tunnels", request).await
-    }
+        let response: TunnelListByRegionResponse = self.execute_json("list_cluster_tunnels", request).await?;
+        let mut tunnels = Vec::new();
+        for region in response.value {
+            tunnels.extend(region.value);
+        }
+        Ok(tunnels)    }
```

Comment:
Run `cargo fmt` to automatically format everything

Reply:
done